c/CMakeLists.txt

```diff
@@ -487,16 +492,18 @@ if (BUILD_STATIC_LIBS)
   target_compile_definitions(qpid-proton-static PUBLIC PROTON_DECLARE_STATIC)
   target_link_libraries (qpid-proton-static
     qpid-proton-core-static
-    $<$<BOOL:${HAS_PROACTOR}>:qpid-proton-proactor-static>
+    ${TARTET_qpid-proton-proactor-static}
```

Comment:
The generator expression bubbled through the ProtonTargets.cmake file (list of libraries to link to) but it was malformed there. I don't exactly remember how and i did not try to solve it. Since the patch is now being seriously considered (something beyond my immediate hopes yesterday), I will revisit this.

Reply:
I have to do it this way. I have to evaluate everything in the context of the Proton project and then see the result in Dispatch (or whatever dependent project).